src/messages/TextWithButtons/TextWithButtons.tsx

```diff
@@ -42,9 +42,9 @@ const TextWithButtons: FC = () => {
 		<div className={`webchat-${classType}-template-root`}>
 			{text && (
 				<Text
+					{...props}
 					content={text}
 					className={`webchat-${classType}-template-header`}
-					id={webchatButtonTemplateTextId}
```

Comment:
Why was this removed? This could potentially break some tests in the webchat repo

Reply:
you are right, I will revert this!